src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs

```diff
@@ -86,7 +86,12 @@ public class Beneficiary : IValidatableObject
     /// The number of distinct authorisers that have authorised the beneficiary.
     /// </summary>
     public int AuthorisersCompletedCount { get; set; }
-    
+
+    /// <summary>
+    /// A list of authentication types allowed to authorise the payout.
+    /// </summary>
+    [CanBeNull] public List<string> AuthenticationMethods { get; set; }

```

Comment:
Is there any reason for this not to be a List of Enum values?

Reply:
Ah no. I just went with strings. I could change to list of enums